spec/features/bulkrax_export_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Bulkrax export', clean: true, perform_enqueued: true do
+  let(:user) { create(:user, email: 'test@example.com') }
+  # let! is needed below to ensure that this user is created for file attachment because this is the depositor in the CSV fixtures
+  let(:depositor) { create(:user, email: 'batchuser@example.com') }
+  let(:exporter) do
+    create(:bulkrax_exporter_worktype,
+           user: user,
+           field_mapping: Bulkrax.field_mappings["HykuAddons::CsvParser"],
+           parser_klass: "HykuAddons::CsvParser",
+           export_source: work.class.to_s,
+           limit: 0)
+  end
+  let(:work) { create(:fully_described_work, user: depositor.email) }
+
+  before do
+    # Make sure default admin set exists
+    AdminSet.find_or_create_default_admin_set_id
+  end
+
+  after do
+    # TODO: cleanup export files and path
+  end
+
+  describe 'export job' do
+    before do
+      work
+    end
+
+    it 'creates csv and zip' do
+      Bulkrax::ExporterJob.perform_now(exporter.id)
+      expect(File.exist?(exporter.parser.setup_export_file)).to eq true
```

Comment:
I think this is more expressive:
```suggestion
      expect(File.exist?(exporter.parser.setup_export_file)).to be_truthy
```